.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 node_js:
-  - "4"
```

Comment:
we still need to test on and support 4 and 6

Reply:
@bhousel since it is just the dev side, is there something which is blocking for just going for node 8.
The reason I am pushing for node 8 and ditching 4, 6 is for being able to to use async await in the build process.

Reply:
I'm not comfortable forcing node 8 on anyone else who wants to contribute to iD.  It's important for this project keeping developer friction low.

Anyway, `npm run build` currently takes 5 seconds.  I don't think we need to optimize this.